lib/madwf_ml.cpp

```diff
@@ -79,17 +84,16 @@ namespace quda
 
       forward_tmp = std::make_unique<ColorSpinorField>(csParam);
       backward_tmp = std::make_unique<ColorSpinorField>(csParam);
+      profile.TPSTOP(QUDA_PROFILE_IO);
```

Comment:
Do we need to restart `QUDA_PROFILE_INIT` here? As in, add `profile.TPSTART(QUDA_PROFILE_INIT);`?

Reply:
We `return` after this? `QUDA_PROFILE_INIT` will be called in the main solver.

Reply:
Ah, I better see what's going on, I overlooked how `QUDA_PROFILE_INIT` was started at the top of this function. Disregard!